webpack.config.demo.js

```diff
@@ -28,7 +28,7 @@ module.exports = {
   externals: {
     'node-fetch': 'fetch',
     'text-encoding': 'TextEncoder',
-    'urlutils': 'URL',
```

Comment:
Is it okay to remove `urlutils`? Dependents might still use it.
It's still in `yarn.lock`.